packages/quick_blue/lib/quick_blue.dart

```diff
@@ -39,6 +39,11 @@ class QuickBlue {
       .map((item) => BlueScanResult.fromMap(item));
   }
 
+  static Stream<AvailabilityState> get availabilityChangeStream {
```

Comment:
Move `get availabilityChangeStream` above `setAvailabilityHandler` and remove `setAvailabilityHandler`

Reply:
This seems not revolved